lib/python/picongpu/plugins/plot_mpl/phase_space_visualizer.py

```diff
@@ -152,7 +152,9 @@ def _create_plt_obj(self, idx):
             va='center', rotation=270,
             transform=self.colorbar_axes[idx].transAxes)
 
-        self.ax.set_xlabel(r'${0}$ [${1}$]'.format(meta.r, r'\mathrm{\mu m}'))
+        self.cbar.set_label(r'$Q / \mathrm{d}r \mathrm{d}p$',
+                            r'[$\mathrm{C s kg^{-1} m^{-2}}$]')
+        self.ax.set_xlabel(r'${0}$ [$\mathrm{\mu m}$]'.format(meta.r))
```

Comment:
this is already done in the current version of the `dev` branch, so please remove.